cmake/Builtins.cmake

```diff
@@ -17,15 +19,39 @@ endmacro()
 
 if(BUILD_VC)
   list(APPEND BUILTINS Vc)
-  add_subdirectory(builtins/Vc)
+  add_subdirectory(cmake/Vc)
```

Comment:
Please don't rename the files, I prefer the structure with separate files as before.